reference/import_export/formats/attribute.rst

```diff
@@ -0,0 +1,39 @@
+Attribute data structure
+========================
+
+Data structure
+--------------
+The import/export file follows this data structure:
+
+- **code** (required): the attribute code.
+- **type** (required): the Akeneo PIM attribute type.
+- **localizable** (required): boolean field (0 or 1) defining if the attribute is localizable or not.
+- **scopable** (required): boolean field (0 or 1) defining if the attribute is scopable or not.
+- **group** (required): the attribute group of the attribute. Default could be "other".
```

Comment:
Default could be "other" => Default value **should**, **is**, **must be** would be better